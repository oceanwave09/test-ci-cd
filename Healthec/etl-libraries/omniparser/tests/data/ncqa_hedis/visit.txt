95000           20220831                         E0433       S79.829S S23.123S                                                                                                                                                                                                                   X            2UNK001    N1 
95000           20221118                99254                S82.846E J43.9    M23.302  S66.500S Q62.32   S70.361A S06.896S                                                                                                                                                                      X            2UNK003    N2 
95000           20210824                99284                S22.040K S82.861C S73.005A S34.132S S85.141D S92.152D E13.3592 S92.222A S83.093A                                                                                                                                                    X            2UNK001    N3 
95000           20220729                99282                E11.3293                                                                                                                                                                                                                            X            2UNK001    N4 
95001           20210305                99281                E13.37X3 S99.821A                                                                                                                                                                                                                   X            2UNK001    N1 