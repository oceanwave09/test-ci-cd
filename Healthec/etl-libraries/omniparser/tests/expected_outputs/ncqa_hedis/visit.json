[
	{
		"claim_id": "1",
		"claim_status": "2",
		"date_of_service": "20220831",
		"hcpcs": "E0433",
		"icd_diagnosis_2": "S23.123S",
		"icd_identifier": "X",
		"member_id": "95000",
		"principal_icd_diagnosis": "S79.829S",
		"provider_id": "UNK001",
		"row_id": "e4df1966-8c9d-4365-b44a-3359d44c3db8",
		"supplemental_data": "N"
	},
	{
		"claim_id": "2",
		"claim_status": "2",
		"cpt": "99254",
		"date_of_service": "20221118",
		"icd_diagnosis_2": "J43.9",
		"icd_diagnosis_3": "M23.302",
		"icd_diagnosis_4": "S66.500S",
		"icd_diagnosis_5": "Q62.32",
		"icd_diagnosis_6": "S70.361A",
		"icd_diagnosis_7": "S06.896S",
		"icd_identifier": "X",
		"member_id": "95000",
		"principal_icd_diagnosis": "S82.846E",
		"provider_id": "UNK003",
		"row_id": "67c7c85f-ba98-4468-84c5-e4e1f13a34eb",
		"supplemental_data": "N"
	},
	{
		"claim_id": "3",
		"claim_status": "2",
		"cpt": "99284",
		"date_of_service": "20210824",
		"icd_diagnosis_2": "S82.861C",
		"icd_diagnosis_3": "S73.005A",
		"icd_diagnosis_4": "S34.132S",
		"icd_diagnosis_5": "S85.141D",
		"icd_diagnosis_6": "S92.152D",
		"icd_diagnosis_7": "E13.3592",
		"icd_diagnosis_8": "S92.222A",
		"icd_diagnosis_9": "S83.093A",
		"icd_identifier": "X",
		"member_id": "95000",
		"principal_icd_diagnosis": "S22.040K",
		"provider_id": "UNK001",
		"row_id": "b05e9942-8fff-45bb-9a8c-62fe5ba1a953",
		"supplemental_data": "N"
	},
	{
		"claim_id": "4",
		"claim_status": "2",
		"cpt": "99282",
		"date_of_service": "20220729",
		"icd_identifier": "X",
		"member_id": "95000",
		"principal_icd_diagnosis": "E11.3293",
		"provider_id": "UNK001",
		"row_id": "1862d3ba-10ab-4d0b-a796-000e98de790d",
		"supplemental_data": "N"
	},
	{
		"claim_id": "1",
		"claim_status": "2",
		"cpt": "99281",
		"date_of_service": "20210305",
		"icd_diagnosis_2": "S99.821A",
		"icd_identifier": "X",
		"member_id": "95001",
		"principal_icd_diagnosis": "E13.37X3",
		"provider_id": "UNK001",
		"row_id": "116a4c04-f7d4-4c25-93a7-1069e99687fc",
		"supplemental_data": "N"
	}
]
